primitives/src/policy.rs

```diff
@@ -6,6 +6,12 @@ use once_cell::sync::OnceCell;
 /// Global policy
 static GLOBAL_POLICY: OnceCell<Policy> = OnceCell::new();
 
+#[cfg_attr(
+    feature = "ts-types",
+    derive(nimiq_serde::Serialize),
+    serde(rename_all = "camelCase"),
```

Comment:
It seems this rename rule has no effect, in the types the properties are still snake_cased.

Reply:
Sleeping over it, I noticed where our confusion may come from.

The serde attributes are only respected by `tsify`, not by `wasm-bindgen`. Unfortunately, wasm-bindgen doesn't have any rule like that ([issue still open](https://github.com/rustwasm/wasm-bindgen/issues/1818)), so you must add a `js_name` rule to all properties instead :/